src/multibody/model.hpp

```diff
@@ -115,22 +115,24 @@ namespace se3
       , njoint(1)
       , nbody(1)
       , nFrames(0)
-      , inertias(1)
-      , jointPlacements(1)
+      , inertias(1, Inertia::Zero())
```

Comment:
NaN or Inf ?

I was told the reason for not initializing the inertia of the world. I agree with the argumentation.